target/linux/airoha/dts/en7581.dtsi

```diff
@@ -163,6 +163,48 @@
 			#reset-cells = <1>;
 		};
 
+		i2cclock: i2cclock@0 {
+			#clock-cells = <0>;
+			compatible = "fixed-clock";
+
+			/* 20 MHz */
+			clock-frequency = <20000000>;
+		};
+
+		i2c0: i2c0@1fbf8000 {
+			compatible = "mediatek,mt7621-i2c";
+			reg = <0x0 0x1fbf8000 0x0 0x100>;
+
+			clocks = <&i2cclock>;
+
+			/* 400 kHz */
+			clock-frequency = <400000>;
+			resets = <&scuclk EN7581_I2C_MASTER_RST>;
+			reset-names = "i2c0";
+
+			#address-cells = <1>;
+			#size-cells = <0>;
+
+			status = "enable";
```

Comment:
@merbanan should it be disabled?

Reply:
Yes, I forgot to fix the dts. For the ref board only i2c0 should be enabled. I'll address this.

Reply:
@merbanan ack, thx. Can you please rebase on top of my branch?